site/docs/configuration.md

```diff
@@ -94,4 +95,5 @@ df.write
 | ------------ | -------------------------- | ------------------------------------------------------------ |
 | write-format | Table write.format.default | File format to use for this write operation; parquet or avro |
 | target-file-size-bytes | As per table property | Overrides this table's write.target-file-size-bytes     |
+| disable-nullability-check | false         | Disables the nullable check on fields                        |
```

Comment:
How about `check-nullability` and default it to `true`?

Reply:
👍